### DISCUSSION TITLE
i got error ImportError: [E048] Can't import language 'en' or any matching language from spacy.lang: No module named "spacy.lang.'en'"

### DISCUSSION THREAD

**Discussion by Dhananjay-kumrawat** [explosion/spaCy#13382] [Help: Coding & Implementations]
i got ImportError: [E048] Can't import language 'en' or any matching language from spacy.lang: No module named "spacy.lang.'en'" error when i work it with Streamlit v1.31.1
i also load nlp= spacy.load('en_core_web_sm') in my code please help me how to solve this error
![Screenshot 2024-03-15 231715](https://github.com/explosion/spaCy/assets/136690100/985c08e5-c424-4ebd-966a-4fcce0f59632)
![Screenshot 2024-03-15 231838](https://github.com/explosion/spaCy/assets/136690100/71ed17ec-4027-4325-93fc-54b3d1b7cc26)

**Comment by svlandeg** [comment]
Hi!

Sorry to hear you're running into trouble. On the surface, this looks like an installation error to me. Could you create a new virtual environment, install your libraries from scratch, share with us the `pip list` output and the full code sample you're trying to run (copy/paste it, don't screenshot, so we can replicate), as well as the full error (including stack trace) you're getting? This will help us get to the bottom of the issue.
